lib/explorer/polars_backend/series.ex

```diff
@@ -616,7 +616,11 @@ defmodule Explorer.PolarsBackend.Series do
 
   @impl true
   def trim(series),
-    do: Shared.apply_series(series, :s_trim)
+    do: Shared.apply_series(series, :s_trim, [""])
```

Comment:
For completeness, we would remove this `trim/1` if we remove the callback.

Reply:
ahh, i see. I'll give that a go